amlip_cpp/src/cpp/node/ParentNode.cpp

```diff
@@ -27,11 +27,11 @@ namespace node {
 
 ParentNode::ParentNode(const char* name, types::NodeKind node_kind)
     : participant_(name)
-    , current_state_(types::StateKind::STOPPED)
-    , node_kind_(node_kind)
     , status_writer_(participant_.create_writer<types::StatusDataType>(
         network::STATUS_TOPIC_NAME,
         network::status_writer_qos()))
+    , current_state_(types::StateKind::STOPPED)
```

Comment:
NIT: should we change all enumerations to lowercase?

Reply:
Yes, and add Custom Enum builder when needed